container_files/tutorials/Loading data from HTTP servers or online file storage services.ipynb

```diff
@@ -0,0 +1,156 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading data with http:// or https://\n",
+    "MLDB makes it very easy to load with the public web. A powerful feature of MLDB allows us to load publicly avaialbe data directly from a web address. The datasets below are from the [Stanford Network Analysis Project (SNAP)](http://snap.stanford.edu/index.html). Here, we have chosen to load [Facebook's Social Circles](http://snap.stanford.edu/data/egonets-Facebook.html) dataset. In our first example, we will be [`importing a text file`](../../../../doc/#builtin/procedures/importtextprocedure.md.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<Response [201]>\n"
+     ]
+    }
+   ],
+   "source": [
+    "dataUrl = \"http://snap.stanford.edu/data/facebook_combined.txt.gz\"\n",
+    "\n",
+    "print mldb.put(\"/v1/procedures/import_data\", {\n",
+    "    \"type\": \"import.text\",\n",
+    "    \"params\": {\n",
+    "        \"dataFileUrl\": dataUrl,\n",
+    "        \"headers\": [\"node\", \"edge\"],\n",
+    "        \"delimiter\": \" \", \n",
+    "        \"quotechar\": \"\",\n",
+    "        \"outputDataset\": \"import_URL1\",\n",
+    "        \"runOnCreation\": True\n",
+    "    }\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the targeted file is inside an archive, we can specify the specific path to the file, as seen in the example below. Here, we load the '3980.circles' file within the 'facebook' folder."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<Response [201]>\n"
+     ]
+    }
+   ],
+   "source": [
+    "dataUrl = \"http://snap.stanford.edu/data/facebook.tar.gz\"\n",
+    "\n",
+    "print mldb.put(\"/v1/procedures/import_data\", {\n",
+    "    \"type\": \"import.text\",\n",
+    "    \"params\": {\n",
+    "        \"dataFileUrl\": \"archive+\" + dataUrl + \"#facebook/3980.circles\",\n",
+    "        \"headers\": [\"circles\"],\n",
+    "        \"delimiter\": \" \", \n",
+    "        \"quotechar\": \"\",\n",
+    "        \"outputDataset\": \"import_URL2\",\n",
+    "        \"runOnCreation\": True\n",
+    "    }\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: you will notice that we have grouped all of our rows in a single column. [Using Javascript inside MLDB's Query Language statements with the built-in jeval function](../../../../ipy/notebooks/_tutorials/_latest/Using%20the%20jseval%20function%20inside%20an%20SQL%20statement.ipynb), we will structure the data in a nicer way."
```

Comment:
Fais un select limit 5 pour monter les données. 

Genre we can now take a look at the data we just imported. [select limit 5]
Ensuite: Notice that all rows are grouped in a single column. In the [nom du tutorial] tutorial, we will show how to use the jseval function to structure the data in a nicer way.